INSTALL

```diff
@@ -479,8 +479,7 @@
   enable-tls1_3
                    TODO(TLS1.3): Make this enabled by default
                    Build support for TLS1.3. Note: This is a WIP feature and
-                   does not currently interoperate with other TLS1.3
-                   implementations! Use with caution!!
+                   may have nasty bugs.  Use with caution!!
```

Comment:
Yes, you are right - the connection should just fall back to TLSv1.2 in those cases. As far as users are concerned though they have two TLSv1.3 implementations that fail to negotiate TLSv1.3 with each other. I think that would be seen as an interop failure. Perhaps just something about it not working with other implementations on different draft versions is all that is required.